uniprot-search/src/main/java/org/uniprot/store/search/document/uniparc/UniParcDocument.java

```diff
@@ -98,6 +101,11 @@ public void setOrganismNames(List<String> organismNames) {
         this.organismNames = new HashSet<>(organismNames);
     }
 
+    @Field("organism_id")
```

Comment:
This needs to be in the Builder, right? because this way you are making document object mutable.

Reply:
Yes, this is the way it works, otherwise Solr was complaining. I just made it work by seeing other fields taxonomy_id and organism_name